app/src/screens/Call/index.tsx

```diff
@@ -55,6 +58,14 @@ const Call: React.FC = () => {
     [dialpad, hangup, mute, selectAudioOutputDevice],
   );
 
+  const navigation = useNavigation<StackNavigationProp<'App'>>();
+
+  useEffect(() => {
+    if (callStatus === TwilioCall.State.Disconnected) {
+      setTimeout(() => navigation.navigate('Dialer'), 1000);
```

Comment:
Why do you need a setTimeout?

Reply:
<img width="246" alt="Screen Shot 2023-03-31 at 9 12 18 AM" src="https://user-images.githubusercontent.com/35968892/229174006-81952e13-52f5-4d93-a8d7-e3970b5d15e1.png">

Without the setTimeout you will not be able to see the "disconnected" UI. I just thought it would be a more user friendly experience. But if you don't think the user needs to see the "disconnected" UI then I will remove the set timeout.